frontend/rolecall/src/app/cast/cast-drag-and-drop.component.html

```diff
@@ -9,30 +9,46 @@
         </mat-icon>
       </button>
     </div>
-    <div *ngFor="let position of data; let i1 = index" class="position-container">
-      <div *ngIf="i1 < positionVals.length">
-        <h3 class="position-title">{{positionVals[i1].name}}</h3>
+    <div *ngFor="let position of castPositions; let i1 = index" class="position-container">
+      <div *ngIf="i1 < castPositions.length">
         <div class="table-container">
           <div class="horizontal-list" id="col-headers">
             <div class="cell cellBothHeader">
+              {{ castPositions[i1].pos.name }}
             </div>
-            <div *ngFor="let header of columnHeaders[i1]" class="cell cellColHeader">
-              {{ header }}
+            <div class="dancer-count-buttons-container">
+              <button mat-icon-button aria-label="Increment Dancer Count Button" class="dancer-count-button"
```

Comment:
Please wrap the attributes and properties one-per-line to make them easier to find/skim through

Reply:
Done.